mkosi.default.d/debian/10-mkosi.debian

```diff
@@ -8,77 +8,63 @@ Distribution=debian
 Release=testing
 
 [Content]
+Packages=
+        cryptsetup-bin
+        iproute2
+        isc-dhcp-server
+        libbpf0
+        libfido2-1
+        libglib2.0-0
+        libgnutls30
+        libidn2-0
+        libmicrohttpd12
+        libp11-kit0
+        libpam0g
+        libpwquality1
+        libqrencode4
+        libtss2-esys-3.0.2-0
```

Comment:
maybe use the -dev for all these libraries, so that it will require less maintenance when there are ABI breaks? extra cost should be minimal, some header files and symlinks, and maybe the odd static lib or two

Reply:
Do they break ABI that often? There were about 7 months between the last two releases. That don't seem like too much churn

Reply:
it's really out of our control, and to my mind the less churn we need to worry about, the better